docs/field-level-encryption.md

```diff
@@ -16,104 +16,6 @@ The resulting document will look similar to the following to a client that doesn
 
 You can read more about CSFLE on the [MongoDB CSFLE documentation](https://www.mongodb.com/docs/manual/core/csfle/) and [this blog post about CSFLE in Node.js](https://www.mongodb.com/developer/languages/javascript/client-side-field-level-encryption-csfle-mongodb-node/).
 
-Note that Mongoose does **not** currently have any Mongoose-specific APIs for CSFLE.
```

Comment:
I moved this to the bottom of the page to emphasize the new Mongoose API for FLE.